src/main/kotlin/io/sip3/salto/ce/sip/SipCallHandler.kt

```diff
@@ -278,12 +282,14 @@ open class SipCallHandler : AbstractVerticle() {
         val attributes = session.attributes
                 .toMutableMap()
                 .apply {
-                    put("state", session.state)
+                    put(Attributes.state, session.state)
+                    if (!recordCallUsersAttributes) {
+                        put(Attributes.caller, "")
+                        put(Attributes.callee, "")
+                    }
                     session.duration?.let { put(Attributes.duration, it) }
                     session.setupTime?.let { put(Attributes.setup_time, it) }
                     session.establishTime?.let { put(Attributes.establish_time, it) }
-                    remove(Attributes.caller)
-                    remove(Attributes.callee)
                     remove(Attributes.state)
```

Comment:
Could you please explain why we need to [put Attributes.state value](https://github.com/sip3io/sip3-salto-ce/pull/7/files#diff-f8d55274b1ab40e6e6f022ffd4156bb1R285) and then remove it?

Reply:
Great catch